### DISCUSSION TITLE
When querying the sidebar you can't click and drag on the XYpad

### DISCUSSION THREAD

**Discussion by marioepsley** [cables-gl/cables_docs#692] [! Welcome - readme first!]
I noticed I can't click and drag on the XYpad when querying the sidebar.
You can do with the sliders.

Here's an example video

https://github.com/cables-gl/cables_docs/assets/36418408/579962e8-630d-46b3-ba9b-3a29d436d001

and a very simplified patch for you to see for yourself.

https://dev.cables.gl/edit/niEFl4

Just hover over the sidebar and try and drag the XYpad point. 
Funnily you can still click it about, just not drag.

**Comment by pandrr** [comment]
added an element output to the operator, then you can use this with the eventlistener op directly
